allennlp/training/trainer.py

```diff
@@ -145,7 +167,7 @@ def _save_checkpoint(self,
         if is_best:
             logger.info("Best validation performance so far. "
                         "Copying weights to %s/best.th'.", self._serialization_prefix)
-            shutil.copy(model_path, os.path.join(model_path, "best.th"))
+            shutil.copyfile(model_path, os.path.join(self._serialization_prefix, "best.th"))
```

Comment:
No - this is for restoring training, which you want to happen from the last saved point. Loading the best model to test on some data is not the use case I had in mind here.